runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -101,6 +101,33 @@ J9::ARM64::TreeEvaluator::generateFillInDataBlockSequenceForUnresolvedField(TR::
    TR_ASSERT_FATAL(false, "This helper implements platform specific code for Fieldwatch, which is currently not supported on ARM64 platforms.\n");
    }
 
+static void wrtbarEvaluator(TR::Node *node, TR::Register *srcReg, TR::Register *dstReg, bool srcNonNull, bool needDeps, TR::CodeGenerator *cg)
```

Comment:
This implementation isn't needed in this PR as it was merged as part of #6495.  Please remove.  That is, assuming they are identical...

Reply:
Removed previously merged function wrtbarEvaluator.